packages/volto/src/helpers/FormValidation/FormValidation.jsx

```diff
@@ -215,6 +215,20 @@ const validateFieldsPerFieldset = (
       );
     }
 
+    let widgetInternalErrors = [];
+    if (widgetName) {
+      const widgetNameValidationCriteria = config.getUtilities({
+        type: 'custom-validator-widget',
```

Comment:
I don't see why we have to give it another name to the type. For me, it's still a validator for a specific widget, handled in the same way as in:

https://github.com/plone/volto/blob/main/packages/volto/src/helpers/FormValidation/FormValidation.jsx#L172-L184

Maybe we have to improve the error checking to cover the return of the validator. I am missing something?